### PR TITLE
ci: add scheduled property-test workflow

### LINKED ISSUE #106: Run property tests in dedicated workflow
## Description

We have a set of property tests that we restrict to shorter test cases because they can holdup PRs and tests locally. 

I want a dedicated workflow to run on a schedule to run the property tests for an extended amount of time. 

If we get a failure, it should open an issue with the results. 

The main goal is to capture specific edge cases from properties and encode them in a unit test.

### PR DESCRIPTION
**Pull request by mkaput**
Add a dedicated GitHub Actions workflow for property tests so they no longer run in the regular CI test job. The new workflow runs daily at 00:00 UTC. It automatically creates an issue when property tests fail (including run metadata and log tail). Using a random seed so scheduled executions explore different generated inputs over time.

Special consideration: automatic issue creation might be noisy. We need to see how this behaves in time.

fix #106

### CODE REVIEW

**Comment by mhanberg** [comment]
Thanks. I don't think it'll generate spam issues, because if we trust the properties, then they are valid things we need to fix.

You need to update the max_cases of the properties to be configurable by an env var.

In CI we'll set it to something really high that can run for a long time. locally we can default it to whatever it currently is.

You can see the bigger values that were shorted in this commit: https://github.com/elixir-tools/spitfire/pull/79/changes/8128f419067758d5747b8c255931fdc00f4b59c2.

They do range, with the biggest being 2mil and the others being like 500k and 250k or less. maybe just settle for running them all at like 250k in ci.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ jobs:
         run: nix develop --command bash -c "mix deps.get"
 
       - name: Unit Tests
-        run: nix develop --command bash -c "mix test"
+        run: nix develop --command bash -c "mix test --exclude property"
```
[the remark this answers is not in the document]
Comment (mkaput) [reply .github/workflows/ci.yaml]: TBH I disagree - this will keep the havoc with proptests spuriously breaking random PR unit tests
